<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REDTabSettingsServices</class>
 <widget class="QWidget" name="REDTabSettingsServices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>436</width>
        <height>424</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QCheckBox" name="chkbox_gpu">
           <property name="text">
            <string>Enable GPU</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="chkbox_webserver">
           <property name="text">
            <string>Start Web Server</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="chkbox_splashscreen">
           <property name="text">
            <string>Show Splash Screen</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_info">
           <property name="text">
            <string>Disabling services can make the system boot faster.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="chkbox_desktopenv">
           <property name="text">
            <string>Start Desktop Environment</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="chkbox_ap">
           <property name="text">
            <string>Access Point Mode</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QPushButton" name="pbutton_services_save">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>chkbox_gpu</tabstop>
  <tabstop>chkbox_desktopenv</tabstop>
  <tabstop>chkbox_webserver</tabstop>
  <tabstop>chkbox_splashscreen</tabstop>
  <tabstop>chkbox_ap</tabstop>
  <tabstop>pbutton_services_save</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
